MapboxNavigation/MapboxSpeechController.swift

```diff
@@ -0,0 +1,160 @@
+import Foundation
+import CoreLocation
+import AVFoundation
+import MapboxSpeech
+import MapboxCoreNavigation
+import MapboxDirections
+
+/**
+ `MapboxVoiceController` extends the default `RouteVoiceController` by providing a more robust speech synthesizer via the Mapbox Speech API. `RouteVoiceController` will be used as a fallback during poor network conditions.
+ */
+@objc(MBMapboxVoiceController)
+public class MapboxVoiceController: RouteVoiceController {
```

Comment:
The file is named MapboxSpeechController.swift, but the class is named MapboxVoiceController.